src/containers/Onboarding/CreateGroup.js

```diff
@@ -40,7 +43,37 @@ class CreateGroup extends Component {
   * @param {Object} fields
   */
   _handleSubmit(fields) {
-    return this.props.createGroup(fields);
+    const promise = new Promise((resolve) => {
+      resolve(this.props.createGroup(fields));
+    });
+
+    const index = {
+      confirmation: 0,
+      delivery: 1
+    };
+
+    promise.then((group) => {
+      const rule = new RRule({
```

Comment:
I found the solution! Someone took over the package and fixed it in a fork https://github.com/jkbrzt/rrule/issues/160#issuecomment-268724051